packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -27,6 +27,8 @@ import AccountRecoveryConfirmKey from '../../pages/ResetPassword/AccountRecovery
 import SignupConfirmed from '../../pages/Signup/SignupConfirmed';
 import ConfirmSignupCode from '../../pages/Signup/ConfirmSignupCode';
 
+import SigninReported from '../../pages/Signin/SigninReported';
```

Comment:
I think your other ticket had the Signin routes above Signup - should this import be moved up for easier merging and nice import grouping?

Reply:
Oh smart thank you!!

Reply:
Probably not worth fixing, but I just noticed sometimes we have `SignInReportedView` above in fxa-content-server and `SigninReported` here in fxa-settings (`SignIn` vs `Signin`).

```sh
git grep -Eih "\bimport Sign(In|Up).*? from " -- ':!packages/fxa-payments-server' | sort -uf | head -15

import Signin, { SigninProps } from '.';
import Signin, { viewName } from '.';
import SigninBounced, { SigninBouncedProps } from '.';
import SigninBounced, { viewName } from '.';
import SignInBouncedView from '../views/sign_in_bounced';
import SigninConfirmed from '.';
import SigninConfirmed, { viewName } from '.';
import SignInMixin from './mixins/signin-mixin';
import SigninMixin from './signin-mixin';
import SignInMixin from 'views/mixins/signin-mixin';
import SignInPasswordView from '../views/sign_in_password';
import SignInPasswordView from 'views/sign_in_password';
import SignInReasons from '../lib/sign-in-reasons';
import SignInReasons from './sign-in-reasons';
import SignInReasons from 'lib/sign-in-reasons';
...
```